### PR TITLE
Single video page and like/dislike

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Looks good to merge

**Review by mnath30 — APPROVED** [review]
Looks good to merge

**Review by MaruthiPothuganti — APPROVED** [review]
CodeLooks Good. PR can be  merged

File: src/Components/Navbar/Navbar.js [hunk 1]
```diff
@@ -10,8 +10,15 @@ import {
...
 import "./Navbar.css";
+// import { useSelector } from "react-redux";
 
 const Navbar = () => {
+  // const {
+  //   user: { token },
+  // } = useSelector((store) => store.auth);
+
+  // const token = localStorage.getItem("token");
+  // console.log(token, typeof token);
```
Comment (ShrutiLakhani) [inline src/Components/Navbar/Navbar.js:21]: Please remove the commented code

File: src/Components/Navbar/Navbar.js [hunk 2]
```diff
@@ -27,6 +34,15 @@ const Navbar = () => {
...
           </Link>
+          {/* {token === null ? (
+            <Link to="/Login" className="link_btn">
+              <Button color="inherit">Login</Button>
+            </Link>
+          ) : (
+            <Link to="/Home" className="link_btn">
+              <Button color="inherit">LogOut</Button>
+            </Link>
+          )} */}
```
Comment (ShrutiLakhani) [inline src/Components/Navbar/Navbar.js:45]: Here as well

**Review by ShrutiLakhani — APPROVED** [review]
Good work on the UI. Code looks good to merge

File: src/Components/Navbar/Navbar.js (hunk 1 above)
Comment (ashishkatiyar2404) [reply src/Components/Navbar/Navbar.js:21]: okay

File: src/Components/Navbar/Navbar.js (hunk 2 above)
Comment (ashishkatiyar2404) [reply src/Components/Navbar/Navbar.js:45]: okay

**Pull request merged**